src/sv_broadcast.c

```diff
@@ -418,13 +418,16 @@ void SVC_Broadcast(void)
 	qbool spectalk;
 	qbool started;
 	qbool valid;
+	char addrport[22];
 	char log[1024];
 	char out[1024];
```

Comment:
Unless it’s unconditionally set, I would recommend taking up the habit of initializing local variables. Especially in big functions like this. Set pointers to NULL, init arrays to 0, etc.